lib/echocat.rb

```diff
@@ -1,7 +1,59 @@
 # frozen_string_literal: true
+# encoding: UTF-8
+
+require 'csv'
+require 'json'
 
 module Echocat
   def self.run
-    puts 'Hello World!'
+    csv_file_paths = [
+      '/workspaces/ruby-kata-1/data/authors.csv', 
```

Comment:
Any reason this isn't inside of the `convert_and_create_json_files` method? Seems a little out of place here.